msal/src/main/java/com/microsoft/identity/client/internal/controllers/BrokerMSALController.java

```diff
@@ -136,31 +143,32 @@ public AcquireTokenResult acquireTokenSilent(MSALAcquireTokenSilentOperationPara
         }
 
         try {
-            //result = service.acquireTokenSilently(getSilentParameters(request));
-
-            // Path is broken by common/#303 - @kreedula & @shoatman to resolve
-            BrokerResult brokerResult = service.acquireTokenSilently(new BrokerRequest());
+            BrokerResult brokerResult = service.acquireTokenSilently(getBrokerRequest(parameters));
+            AcquireTokenResult acquireTokenResult = new AcquireTokenResult();
+            acquireTokenResult.setTokenResult(brokerResult);
```

Comment:
This line is throwing a build error - it looks like the submodule pointer needs to be updated for this PR

```
:msal:compileLocalDebugJavaWithJavac/home/travis/build/AzureAD/microsoft-authentication-library-for-android/msal/src/main/java/com/microsoft/identity/client/internal/controllers/BrokerMSALController.java:148: error: incompatible types: BrokerResult cannot be converted to TokenResult
            acquireTokenResult.setTokenResult(brokerResult);
                                              ^
Note: /home/travis/build/AzureAD/microsoft-authentication-library-for-android/msal/src/main/java/com/microsoft/identity/client/internal/telemetry/PlatformIdHelper.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
Note: Some messages have been simplified; recompile with -Xdiags:verbose to get full output
1 error
 FAILED
```